silk/tests/silk_run.py

```diff
@@ -198,5 +198,5 @@ def run(self):
 
 
 if __name__ == "__main__":
-
+    sys.argv = ['tests/silk_run.py', '-v2', '-c', CONFIG_PATH+'hwconfig.ini', '-d', RESULT_LOG_PATH,  'ot_test_form_network.py']
```

Comment:
Please remove or comment out this line which is for  debug run

Reply:
I did comment, so we have one line to keep the example, and use it for debugging purpose for future